demosplan/DemosPlanCoreBundle/Entity/Statement/ViewStatementStatus.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * This file is part of the package demosplan.
+ *
+ * (c) 2010-present DEMOS plan GmbH, for more information see the license file.
+ *
+ * All rights reserved
+ */
+
+namespace demosplan\DemosPlanCoreBundle\Entity\Statement;
+
+use DemosEurope\DemosplanAddon\Contracts\Entities\ViewStatementStatusInterface;
+use demosplan\DemosPlanCoreBundle\Entity\CoreEntity;
+use Doctrine\ORM\Mapping as ORM;
+
+class ViewStatementStatus extends CoreEntity implements ViewStatementStatusInterface
+{
+    /**
+     * @ORM\Id()
+     *
+     * @ORM\Column(type="string", length=255)
+     */
+    private string $statement;
```

Comment:
i'm not sure if i understand this. this is the status id right? i mean if we need the statement id (what make sense) than it has to be an extra property and may be unique too